BZ-2025586:Documentation update to reflect changes for capabilities getting dropped when using keyword  in the container's securityContext (#39124)
openshift/openshift-docs: modules/nodes-pods-autoscaling-creating-memory.adoc

> = Creating a horizontal pod autoscaler object for memory utilization by using the CLI

Reviewer: ```suggestion
[id="nodes-pods-autoscaling-creating-memory_{context}"]
= Creating a horizontal pod autoscaler object for memory utilization by using the CLI
```

Reviewer: The extra space between the anchor ID might continue to work, but I suggest removing the extra space while you are in this file.

Using the {product-title} CLI, you can create a horizontal pod autoscaler (HPA) to automatically scale an existing 
`Deployment`, `DeploymentConfig`, `ReplicaSet`, `ReplicationController`, or `StatefulSet` object. The HPA  
scales the pods associated with that object to maintain the average memory utilization you specify, either a direct value or a percentage 
of requested memory.

[NOTE]
====

> It is recommended to to use a Deployment or ReplicaSet objects unless you need a 

Reviewer 2: ```suggestion
It is recommended to not use a `Deployment` or `ReplicaSet` object unless you need a 
```
(same as the other instance of this note)

Author: Same

<1> Use the `autoscaling/v2beta2` API.
<2> Specify a name for this horizontal pod autoscaler object.
<3> Specify the API version of the object to scale:
* For a `Deployment`, `ReplicaSet`, or `Statefulset` object, use `apps/v1`.
* For a `ReplicationController`, use `v1`.
* For a `DeploymentConfig`, use `apps.openshift.io/v1`.

> <4> Specify the type of object. The object must be a `Deployment`, `DeploymentConfig`/`dc`, 
> `ReplicaSet`/`rs`, `ReplicationController`/`rc`, or `StatefulSet`.

Reviewer 2: I have no idea if this is correct or not, but maybe consider:
```suggestion
<4> Specify the type of object. The object must be a `Deployment`, `DeploymentConfig` (`dc`), 
`ReplicaSet` (`rs`), `ReplicationController` (`rc`), or `StatefulSet`.
```

<1> Use the `autoscaling/v2beta2` API.
<2> Specify a name for this horizontal pod autoscaler object.
<3> Specify the API version of the object to scale:
* For a ReplicationController, use `v1`.
* For a DeploymentConfig, use `apps.openshift.io/v1`.
* For a Deployment, ReplicaSet, Statefulset object, use `apps/v1`.
<4> Specify the type of object. The object must be a `Deployment`, `DeploymentConfig`/`dc`, 

> `ReplicaSet`/`rs`, `ReplicationController`/`rc`, or `StatefulSet`.

Reviewer 2: Same comment as above about () vs /
